test/src/main/java/org/apache/accumulo/test/functional/ScanIdIT.java

```diff
@@ -316,7 +315,12 @@ private void addSplits(final AccumuloClient client, final String tableName) {
 
       client.tableOperations().offline(tableName, true);
 
-      sleepUninterruptibly(2, TimeUnit.SECONDS);
+      try {
+        Thread.sleep(SECONDS.toMillis(2));
+      } catch (InterruptedException ex) {
+        Thread.currentThread().interrupt();
+        throw new IllegalStateException("interrupted during sleep", ex);
+      }
```

Comment:
Could just let the exception fall through to the test method instead of trying to catch it and rethrow as another type. It makes it a bit more readable for the test method.

Reply:
Modified in 4ae251cf0a